src/main/java/Daniela_Chihai/com/Endava/SoA5/Java_Basics/Task1.java

```diff
@@ -0,0 +1,15 @@
+package Daniela_Chihai.com.Endava.SoA5.Java_Basics;
+
+import java.util.Scanner;
+
+public class Task1 {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter the number of seconds: ");
+        int nrOfSeconds = scanner.nextInt();  //we enter a number
+        int hours = nrOfSeconds / 3600;         //divides our seconds into nr. of hours
+        int minutes = (nrOfSeconds % 3600) / 60; //divides our seconds into nr. of minutes Left from that our
+        int seconds = (nrOfSeconds % 3600) % 60; //divides our seconds into nr. of seconds Left from our our
```

Comment:
I think you can calculate the seconds left just by this formula: nrOfSeconds % 60

Reply:
Done, thank you.